wgsl/extract-grammar.py

```diff
@@ -1182,37 +1184,50 @@ def flow_build(options):
     # subprocess.run(["npx", "tree-sitter-cli@" + value_from_dotenv("NPM_TREE_SITTER_CLI_VERSION"), "build-wasm", "--docker"],
     #                cwd=options.grammar_dir, check=True)
 
+    def build_library(output_path, input_files):
+        # The py-tree-sitter build_library method with C++17 flags
+        """
```

Comment:
Thank you for adding method comments!  :-)

Reply:
@dneto0 certainly makes it easier to come back to code after a while 😆...